### DISCUSSION TITLE
Error showing

### DISCUSSION THREAD

**Discussion by SuperUser-exe** [AntlerVC/firetable#1388] [Support / Q&A]
Once done all steps showing error, how can ifix this please find below the error message

Oops, something went wrong
This issue has been reported to the Rowy team to be fixed.

ROWY TRACING ID: ROWYERR-A358-DA1D-FB3F-5475E5F5C7E6

![image](https://github.com/rowyio/rowy/assets/26677550/384f0349-b720-4392-9f89-0f24014b8b6c)

**Comment by harinij** [comment]
Could you please share some screenshot/recording with explanation of where you see the issue. It is unclear for us without reproduction steps how to further proceed here. Closing this for now, in case you still facing this issue, reopen by commenting with further detailed explanation including reproduction steps of the issue.

**Reply by SuperUser-exe** [reply]
My requirement is WhatsApp business bot, when customer sending message to my WhatsApp number CHAT GPT need to auto reply to customer and my WhatsApp number Facebook not verified but Twilio verified and i can see the number everything updated when i check anther WhatsApp number i can see profile pic and all the details , because the number purchased from Twilio for customer support we need to use the same number for WhatsApp but Facebook not verifying they rejecting so my first question is without Facebook verification with Twilio can i use this service if possible please need your help for this error
